app/api/test-slack/route.ts

```diff
@@ -129,42 +146,81 @@ export async function POST(request: Request) {
       );
     }
 
-    let webhookResult;
+    let responseText;
```

Comment:
Ok so Vercel actually doesn't have great support of serverless functions calling each other over http like this.

Given that I may separate this preview environment stuff into a separate project one day, let's do this:

- create a new utils function called internalFetch, which has the same api as fetch
- internalFetch should dynamically resolve the POST module in the project and invoke the method directly
- invoke that here